elide-annotations/src/main/java/com/yahoo/elide/security/ChangeSpec.java

```diff
@@ -21,9 +21,17 @@
     @Override
     public String toString() {
         return String.format("ChangeSpec { resource=%s, field=%s, original=%s, modified=%s}",
-                resource,
+                safe(resource),
                 fieldName,
-                original,
-                modified);
+                safe(original),
+                safe(modified));
+    }
+
+    private String safe(Object object) {
```

Comment:
This fixed a possible NPE in `buildErrorResponse` after the datastore transaction closed.
Should the function instead move to a utility class?
```
org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: at org.hibernate.collection.PersistentSet.toString(PersistentSet.java:332) 
at com.yahoo.elide.security.ChangeSpec.toString(ChangeSpec.java:23) 
at com.yahoo.elide.security.permissions.PermissionCondition.toString(PermissionCondition.java:112) 
at com.yahoo.elide.security.permissions.expressions.SpecificFieldExpression.toString(SpecificFieldExpression.java:53) 
at com.yahoo.elide.core.exceptions.ForbiddenAccessException.lambda$new$0(ForbiddenAccessException.java:30) 
at java.util.Optional.map(Optional.java:215) 
at com.yahoo.elide.core.exceptions.HttpStatusException.getVerboseMessage(HttpStatusException.java:71) 
at com.yahoo.elide.core.exceptions.HttpStatusException.getVerboseErrorResponse(HttpStatusException.java:60) 
at com.yahoo.elide.Elide.buildErrorResponse(Elide.java:303) 
at com.yahoo.elide.Elide.handleRequest(Elide.java:237) 
```